packages/datadog-instrumentations/src/jest.js

```diff
@@ -338,7 +338,7 @@ function applySuiteSkipping (originalTests, rootDir, frameworkVersion) {
   numSkippedSuites = jestSuitesToRun.skippedSuites.length
 
   itrSkippedSuitesCh.publish({ skippedSuites: jestSuitesToRun.skippedSuites, frameworkVersion })
-  skippableSuites = []
```

Comment:
what I considered a nice clean up turned out to be a bug 😆. Cleaning up is not very useful anyway, as the project is short lived anyway